### PR TITLE
Some modifications for error handling

### PR DESCRIPTION
**Pull request by nikhil-achole**

### CODE REVIEW

File: tempest/api/workloadmgr/functional/test_functional.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def multiple_vms(self, vm_count, key_pairs, sec_groups):
                 vm_id = self.create_vm(security_group_id=random.choice(sec_groups), key_pair=kptouple[1], key_name=kptouple[0], vm_cleanup=True)
                 vms[vm_id] = [kptouple[0]]
             else:
-                boot_volume_id = self.create_volume(image_id=CONF.compute.image_ref, volume_cleanup=True)
+                boot_volume_id = self.create_volume(size=tvaultconf.bootfromvol_vol_size, image_id=CONF.compute.image_ref, volume_cleanup=True)
```
Comment (savitha-p) [inline tempest/api/workloadmgr/functional/test_functional.py]: Make the default size as CONF.volume.volume_size . Wherever required, you can pass tvaultconf.bootfromvol_vol_size

File: tempest/api/workloadmgr/retention/test_retention.py [hunk 2]
```diff
@@ -66,17 +66,17 @@ def test_retention(self):
...
                 reporting.add_test_step("Retention", tvaultconf.PASS)
                 LOG.debug("Retention worked!!")
             else:
                 reporting.add_test_step("Retention", tvaultconf.FAIL)
                 LOG.debug("Retention didn't work!!")
                 raise Exception("Retention failed")
             if (tvaultconf.cleanup == True):
-                for each in sl:
-                    self.addCleanup(self.snapshot_delete,workload_id, each)
+                for sanpshot in snapshotlist:
```
Comment (savitha-p) [inline tempest/api/workloadmgr/retention/test_retention.py]: Correct the typo.

File: tempest/api/workloadmgr/functional/test_functional.py (hunk 1 above)
Comment (nikhil-achole) [reply tempest/api/workloadmgr/functional/test_functional.py]: yes this is according to that only i think... in base.py for create _volume method volume_size is taken as CONF.volume.volume_size only.... here i need bootfromvol_vol_size so used it

Comment (savitha-p) [reply tempest/api/workloadmgr/functional/test_functional.py]: Okay.

**Pull request merged**